fish_prompt.fish

```diff
@@ -777,7 +777,7 @@ function __bobthefish_prompt_rubies -S -d 'Display current Ruby information'
     else if type -q chruby # chruby is implemented as a function, so omitting the -f is intentional
         set ruby_version $RUBY_VERSION
     else if type -fq asdf
-        asdf current ruby 2>/dev/null | read -l asdf_ruby_version asdf_provenance
+        asdf current ruby 2>/dev/null | read -l -t asdf_ruby_version asdf_provenance
```

Comment:
I'm curious why you didn't go with `string trim`?

Changing `read` to interpret the asdf output as escaped shell strings will likely bite us later.

Reply:
Oh, checked the output and I understand why trim wouldn't have worked. Lemme see if I can come up with something better.